service/src/main/resources/application.yml

```diff
@@ -98,5 +98,8 @@ management:
     health:
       enabled: true
       probes.enabled: true
+      group:
+        liveness:
+          include: livenessState,hybridConnectionListenerHealth
```

Comment:
the liveness probe currently returns the abbreviated `{"status": "UP"}` or `{"status": "DOWN"}`. We could add `show-details: ALWAYS` here so it shows the individual statuses of each of the `livenessState` and `hybridConnectionListenerHealth` subsytems.